docs/source/tutorials/fitting-CBVs.rst

```diff
@@ -0,0 +1,181 @@
+
+Example 5: Fitting CBVs to Remove Long Term Trends
+==================================================
+
+What are CBVs
+~~~~~~~~~~~~~
+
+Cotrending Basis Vectors are used to remove long term trends in light
+curves. They are built from the most common trends observed in each
+channel. You can read more about CBVs in `Demystifying Kepler
+Data <https://arxiv.org/pdf/1207.3093.pdf>`__. They can be used to clean
+lightcurves of common trends experience by all targets.
```

Comment:
We need to explain CBVs in a lot more detail here.  Let's borrow content from Sections 4.2 and 5 of the "Demystifying Kepler Data" (it is fine to copy verbatim from that paper).